.github/CONTRIBUTING.md

```diff
@@ -17,6 +17,17 @@ please file an issue that illustrates the bug with a minimal [reprex](https://ww
 (this will also help you write a unit test, if needed).
 See the tidyverse guide on [how to create a great issue](https://code-review.tidyverse.org/issues/) for more advice.
 
+### Dependencies installation
+
+To install all dependencies of igraph, including dependencies needed for development, call:
```

Comment:
Do these need to be run when the current directory is igraph's source directory? Could you make this clear?